### ISSUE TITLE
Implement Relic cards for Artifacts Expansion

### ISSUE THREAD

**Issue by ultimatefiend** [micahstairs/bga-innovation#184]
This issue will track the implementation of all Relic cards for the Artifacts expansion. Please update this comment before working on any of these cards (if the work is unblocked) so that we don't have duplicate work being done. We no longer need to file a separate issue for each card, since a consolidated list of the cards will keep us more organized.

  - [ ] **215 - Timbuktu**
    - Status: BLOCKED
    - Needs Cities Expansion
  - [ ] **216 - Complex Numbers**
    - Status: NOT STARTED (medium)
  - [ ] **217 - Newton-Wickins Telescope**
    - Status: NOT STARTED (medium)
  - [ ] **218 - Ching Shih**
     - Status: BLOCKED
     - Needs Figures expansion
  - [ ] **219 - Safety Pin**
     - Status: BLOCKED
     - Needs Echoes expansion

**Comment by micahstairs** [comment]
Thanks for filing this! I removed 3 of the relics from this issue to avoid confusion.

**Comment by ultimatefiend** [comment]
Did you create an issue for the other expansions so we don't forget?

**Comment by micahstairs** [comment]
No, but I have a personal list on my phone to remind me that we need to consider relics when it's time to implement them. I'd prefer not to clutter up our list of tasks for now!

**Comment by ultimatefiend** [comment]
is complex numbers unblocked?

**Comment by micahstairs** [comment]
Oh sorry, it's still blocked. I re-opened that other issue. I accidentally closed it early.
